packages/survey/tests/Question/ItemGroup.test.tsx

```diff
@@ -0,0 +1,124 @@
+import { fireEvent, render, testA11y } from '@easyfeedback/test-utils'
+
+import { ItemGroup, ItemGroupProps } from '../../src'
+
+const mockDefaultProps: ItemGroupProps = {
+  name: 'group_name',
+  itemType: 'checkbox',
+  activeColor: '#990033',
+  options: [
+    {
+      value: 'fox_dog',
+      imageSrc:
+        'https://images.unsplash.com/photo-1462953491269-9aff00919695?auto=format&fit=crop&w=634&q=80',
+      label: 'The quick brown fox jumps over the lazy dog',
+    },
+    {
+      value: 'lorem',
+      imageSrc:
+        'https://images.unsplash.com/photo-1623776051205-ab4b9e13e8b0?auto=format&fit=crop&w=659&q=80',
+      label:
+        'Enim inventore aliquid neque voluptates. Ut dolorem saepe explicabo et eum beatae. Autem at voluptas aspernatur possimus consequatur. Eius inventore commodi et in perspiciatis inventore laborum. Beatae eius animi et facere suscipit laboriosam quaerat qui. Assumenda dolores ducimus modi earum rem qui illum.',
+    },
+    {
+      value: 'value_3',
+      imageSrc: '',
+      label: 'Value 3',
+    },
+    {
+      value: 'value_4',
+      imageSrc:
+        'https://images.unsplash.com/photo-1532009877282-3340270e0529?&auto=format&fit=crop&w=1050&q=80',
+      label: 'Value 4',
+    },
+  ],
+  direction: 'vertical',
+  withLabels: false,
+  withImages: false,
+}
```

Comment:
Like before in the story, I think, that the `mockDefaultProps` should only contain parameters that are required.
This could improve our test cases.

Reply:
done!